### PR TITLE
refactor!: return an object instead of throw error when timeout exceeded

### PR DESCRIPTION
**Pull request by hamed-musallam**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -40,16 +40,22 @@ export class Conrec {
     this.hasMinMax = false;
   }
 
+  /**
+   * @typedef {Object} Output
+   * @property {any} contours
+   * @property {boolean} timeout - Whether contour generation finishes in the specified timeout or not
```
Comment (targos) [inline src/index.js]: Doesn't it return the opposite?

Comment (hamed-musallam) [reply src/index.js]: I need just to negate the statement

Comment (hamed-musallam) [reply src/index.js]: What about changing the statement to "Whether contours generation does not finish in the specified timeout or it does in time" is it make things clear or not?

do you have any suggestions?

Comment (targos) [reply src/index.js]: I'd write something like "Whether contour generation had to stop early because it reached the timeout".

**Pull request merged**